package.json

```diff
@@ -30,13 +30,14 @@
     "eslint-loader": "^3.0.3",
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-react": "^7.29.3",
+    "happy-dom": "^10.5.2",
     "husky": "^7.0.4",
-    "jest": "^27.5.1",
     "jest-styled-components": "^7.0.8",
     "lerna": "^5.0.0",
     "nx": "^14.1.9",
     "react": "^18.0.0",
-    "react-dom": "^18.0.0"
+    "react-dom": "^18.0.0",
+    "vitest": "^0.33.0"
```

Comment:
Now that we are using vitest 🚀, we can remove the `babel-jest` and `@types/jest` from our packages, right?

Reply:
I ended up removing some other dependencies as well
You can check in [this commit](https://github.com/React95/React95/pull/406/commits/ed1fad2bb339fdad6948ace46dc975e56c85da5a)